### ISSUE TITLE
Models only running on a single CPU core? (keras_06.ipynb search_space)

### ISSUE THREAD

**Issue by mattjshannon** [mattjshannon/swsnet#7]
I noticed this when iterating over sequential tf.keras models to determine good/ideal training/test splits. See https://github.com/mattjshannon/swsnet/blob/determine-training-size/ipy_notebooks/keras_v06.ipynb, particularly `run_matrix = [run_NN(features, labels, test_size=x) for x in search_space_full]`.

Could just be a limitation of the way the model is constructed, or perhaps tensorflow configuration is needed [(tf.ConfigProto)](https://github.com/tensorflow/tensorflow/blob/r1.10/tensorflow/core/protobuf/config.proto)?
